### DISCUSSION TITLE
Has anyone tried to host it on repl.it?

### DISCUSSION THREAD

**Discussion by FatGrizzly** [Cog-Creators/Red-DiscordBot#4573] [Q&A] [answered]
Title says it

**Comment by yamikaitou** [comment]
Yes and they have been largely unsuccessful. Red is not designed to be ran on platforms like repl.it and sometimes can't due to wording in the ToS.

**Comment by CorruptedAI** [comment]
Yes you can host on replit, you must create a virtual env instance and then install it that way. It works fine for me but there isn't 100 percent uptime

**Comment by Kowlin** [comment, marked as answer]
Despite what anyone says here, We do not support repl.it or any other non-traditional VPS based host. You're welcome to try and host Red on those platforms. However, we will not provide support for these platforms. And we can also not ensure that all of the functionality of Red will work on these types of host.

For the best experience please refer to our hosting list found [here](https://docs.discord.red/en/stable/host-list.html).
